integration_tests/integration_tests/tests/conftest.py

```diff
@@ -8,6 +8,12 @@ def pytest_addoption(parser):
     parser.addoption("--target", action="store", default="postgres")
 
 
+def pytest_configure(config):
```

Comment:
Isn't it better to [use `pytest.ini`](https://docs.pytest.org/en/latest/how-to/mark.html#registering-marks) for that?

Reply:
Yeah, I'll change